packages/jest-cli/src/lib/highlight.js

```diff
@@ -17,6 +17,7 @@ const hit = chalk.reset;
 const miss = chalk.dim;
 
 const trim = '...';
+const relativePathHead = './';
```

Comment:
I'm not sure how this will play with Windows.. maybe we need to use path.sep instead of `/`